errors.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Error responses
```

Comment:
I feel like we should have a more generic `Response codes and errors` site, which includes 200, 201 etc as well.